pkgs/applications/misc/blender/default.nix

```diff
@@ -142,23 +142,20 @@ stdenv.mkDerivation rec {
   blenderExecutable =
     placeholder "out" + (if stdenv.isDarwin then "/Applications/Blender.app/Contents/MacOS/Blender" else "/bin/blender");
   postInstall = lib.optionalString stdenv.isDarwin ''
-    mkdir $out/Applications
-    mv $out/Blender.app $out/Applications
+    mkdir "$out/Applications"
+    mv "$out/Blender.app" "$out/Applications"
   '' + ''
     buildPythonPath "$pythonPath"
-    wrapProgram $blenderExecutable \
+    wrapProgram "$blenderExecutable" \
       --prefix PATH : $program_PATH \
       --prefix PYTHONPATH : "$program_PYTHONPATH" \
       --add-flags '--python-use-system-env'
   '';
 
-  # Set RUNPATH so that libcuda and libnvrtc in /run/opengl-driver(-32)/lib can be
-  # found. See the explanation in libglvnd.
+  # Set RUNPATH so that libcuda and libnvrtc in /run/opengl-driver(-32)/lib
+  # can be found. See the explanation in libglvnd.
```

Comment:
Also it is a good idea to keep unrelated changes to a minimum when doing treewide changes to reduce the risk of merge conflicts and to keep the size of the commit to a minimum. Both the quoting changes in the files and reflowing is very unrelated to the treewide changes done here.